### ISSUE TITLE
Horaire Openhack

### ISSUE THREAD

**Issue by j15e** [webaquebec/openhack#15]
Je suggère quelque chose comme ceci (ça ressemble pas mal à ce qui a été annoncé sur Capitale Ouverte) pour annoncer un horaire complet mixte OpenHack + Hackaton :
### Samedi
- 8h00 : Accueil et déjeuner
- Présentation sympathique des projets et organisation des équipes.
- 9h00 : Présentation et lancement officiel du Hackaton
- 9h30 : Atelier Github 101
- 10h00 : Atelier Rubygems 101
- 12h00 : Diner
- 17h00 : Souper
### Dimanche
- 8h00 : Accueil et déjeuner
- 9h30 : Atelier Travis-CI 101
- 12h00 : Diner
- 16h00 : Présentation des réalisations et projets
- 17h00 : Clôture et remise des prix

**Comment by jipiboily** [comment]
Le samedi, souper? Sur Eventbrite ça disait que ça terminait à 16h. Quel est le plan, ça continue le soir?

**Comment by j15e** [comment]
On veut inciter le gens à rester un peu plus tard vu le Hackaton et être smath en incluant un souper, mais je n'ai pas de plan pour que ça dur toute la nuit ;p

**Comment by jipiboily** [comment]
oki doki, merci de l'info :)

Je prépare une PR pour ajouter un projet...

**Comment by j15e** [comment]
Mais je crois (sous toute réserve de confirmer avec E400 et quelques personnes concernées) de tenir un 5-7 geek samedi soir jusqu'à 20-21h.

**Comment by remi** [comment]
:+1:

**Comment by remi** [comment]
Heu, l’atelier Rubygems 101 là, je ne suis pas certain que ça mérite sa place dans l’horaire. Je pensais juste faire un petit briefing s’il y a des gens qui veulent faire des gems. Moi mon projet n’est pas une gem :stuck_out_tongue:

**Comment by j15e** [comment]
Pareillement pour mon atelier, mais je me disais, aussi bien annoncer un moment et prendre un petit groupe d'intéressés pour leur donner un mini crash-course dès le début (si c'est pertinent) sur la matière. Dit comme ça c'est moins engageant?

**Comment by remi** [comment]
Alright!

**Comment by jipiboily** [comment]
Perso je ne serai pas là le dimanche, déjà que le samedi est un peu un tour de force pour ma part! ;)

**Comment by j15e** [comment]
@jpsirois aucun problème que tu sois présent seulement l'une des deux journées, tu peux même l'indiquer à ton inscription.

**Comment by jpsirois** [comment]
@j15e On peux close ce issue svp?

Je fais du ménage dans mon GitHub et c'est la dernière remaining à enlever hehe.

<img width="946" height="170" alt="Image" src="https://github.com/user-attachments/assets/beef26ed-b2e1-414d-9b21-95122ecae942" />

**Comment by remi** [comment]
@jpsirois En plus, tu t’es retrouvé dans l’issue malgré toi à cause d’un mauvais tagging de JP par @j15e 😂

**Comment by jpsirois** [comment]
Ah non, j'avais bel et bien un commentaire où je posais une question et je l'ai effacé tantôt en me disant que ça l'enlèverait de ma liste d'open Issues, mais c'est la mention qui le garde finalement hehe.

**Comment by remi** [comment]
Et si tu _unsubscribes_ de l’issue?

<img width="316" height="218" alt="" src="https://github.com/user-attachments/assets/eae205c9-7778-4dc4-adbc-776425422ec5" />

**Comment by jpsirois** [comment]
Non, ça ne fonctionne pas non plus, et j'ai même essayé tous les notification settings hehe. Si je me fie au message le mentionne ici, c'est vraiment le mention qui a le plus de weight hehe.

<img width="469" height="420" alt="Image" src="https://github.com/user-attachments/assets/73de99d7-9fe1-4331-9484-ee07dbd93f0a" />

**Comment by j15e** [comment]
Hey, ca fait longtemps! 
Bonne journée tout le monde 👋

**Comment by jpsirois** [comment]
🎉 😎

<img width="938" height="134" alt="Image" src="https://github.com/user-attachments/assets/204fb2a0-d01f-4c32-a7e0-fda5d3b42179" />
